src/components/LoginForm.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
```

Comment:
You shouldn't have `LoginForm` changes as your changes, please `cherry pick` your commits. This article can help you do that: https://www.clock.co.uk/insight/deleting-a-git-commit